addon/components/frost-date-picker.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember'
 
-export default Ember.Component.extend({
+const {Component} = Ember
```

Comment:
@sglanzer no, the only note was the email you sent out yesterday. Didn't see any discussion in #frost.